src/command/game/network.js

```diff
@@ -0,0 +1,27 @@
+const getBroadcastNetworks = televisionNetworks => {
+  let homeTeamNetwork = televisionNetworks.filter(
```

Comment:
Sorry for the late review. I celebrated Chinese new year last week, and I get back to work today.

I prefer to use `ramda` to do the same thing like `array.filter()`. Check [R.find](http://ramdajs.com/docs/#find) 😄

Reply:
@xxhomey19 went ahead and made that change. Thank you for pointing that out the Ramda library is pretty cool going to check it out some more. Let me know if you think there should be any other changes.